entgql/internal/todoplugin/cmd/entgqlgen.go

```diff
@@ -35,7 +36,14 @@ func main() {
 	}
 
 	exEntGQL, err := entgql.NewExtension(
+		entgql.WithConfigPath("./gqlgen.yml"),
+		entgql.WithWhereFilters(true),
 		entgql.WithSchemaGenerator(),
+		entgql.WithSchemaHook(func(g *gen.Graph, s *ast.Schema) error {
+			return nil
+		}),
```

Comment:
hello @crossworth, what do you think about this hook? Is it satisfied with this comment? https://github.com/ent/contrib/pull/267#issuecomment-1077900139

Reply:
Hello @giautm great work!
I have made a few small (and fast) tests and it worked without problems, I think this adds a lot of possibilities.

Maybe in the future we can collect the most common usages for schema hooks and add "helper functions".
Like adding/removing a field/directive, this should make even easier to the consumers.

I can't wait to use all this new features that you guys are adding.